src/PreValidatableRuleInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+use Closure;
+
+interface PreValidatableRuleInterface
```

Comment:
What is the purpose of this interface? It isn't clear from the name. I suggest either improving the name or adding phpdoc.